configuration/pih/petl/jobs/sql/extractions/pmtct_infant_delivery.sql

```diff
@@ -0,0 +1,123 @@
+set sql_safe_updates = 0;
+set session group_concat_max_len = 100000;
+
+select program_id into @eidProgram from program where uuid = '7e06bf82-9f1a-4218-b68f-823082ef519b';
+select encounter_type_id into @labResultsEnc from encounter_type where uuid = '4d77916a-0620-11e5-a6c0-1697f925ec7b';
+select name into @hivDispEncName from encounter_type where uuid = 'cc1720c9-3e4c-4fa8-a7ec-40eeaad1958c';
+
+drop temporary table if exists temp_eid;
+
+create temporary table temp_eid
+(
+patient_id int,
+infant_age_in_months int,
+age_cat_infant varchar(50),
+eid_date_enrolled	datetime,
+eid_date_completed	datetime,
+mom_hiv_emr_id int,
+mom_age_at_delivery int,
+delivery_date date,
+Delivery_with_partograph boolean,
+delivery_location_cat  varchar(255),
+maternity_clinic_type  varchar(255),
+breastfeeding_status  varchar(255),
+hiv_rapid_test_obs int(11),
+hiv_rapid_test_date date,
+hiv_test_result varchar(255),
+pcr_test_obs int(11),
+pcr_test_date date,
+pcr_result varchar(255),
+art_start_date date,
+PMTCT_enrollment_date date,
+is_active_on_art boolean,
+next_dispensing_date date,
+is_LTFU boolean,
+is_exposed boolean,
+is_lab_specimen_rejected boolean,
+health_facility varchar(255)
+);
+
+insert into temp_eid (patient_id,eid_date_enrolled,eid_date_completed)
+select patient_id,date_enrolled,date_completed 
+from patient_program pp 
+where program_id = @eidProgram
+and voided = 0
+;
+
+-- current age, category
+update temp_eid t 
+set infant_age_in_months = current_age_in_months(patient_id);
+
+update temp_eid t 
+set age_cat_infant = 
+case 
+	when infant_age_in_months<2 then '<2 mo'
+	when infant_age_in_months>=2 and infant_age_in_months<=12 then '2-12 mo'
```

Comment:
Are you sure this is meant to be "<=12", and not "<12"?

Reply:
I was assuming that '<2 mo' means strictly less than 2 months, while '2-12 mo' means 2-12 inclusive (i.e. exactly 2 months to 12 months and 29 days or whatever).  Will highlight for Julia when she tests this.